contracts/RewardsHelper.sol

```diff
@@ -54,6 +54,29 @@ contract RewardsHelper is Initializable, OwnableUpgradeable {
         }
     }
 
+    function updateIndexerStatus(address indexer) public {
+        RewardsDistributer rewardsDistributer = RewardsDistributer(settings.getRewardsDistributer());
+        IEraManager eraManager = IEraManager(settings.getEraManager());
+        uint256 currentEra =  eraManager.safeUpdateAndGetEra();
+        if(rewardsDistributer.getLastSettledEra(indexer) >= rewardsDistributer.getRewardInfo(indexer).lastClaimEra){
+            rewardsDistributer.collectAndDistributeRewards(indexer);
+        }
+        //apply all stakers' change of an indexer
+        while(rewardsDistributer.getPendingStakeChangeLength(indexer) > 0){
```

Comment:
Use `getPendingStakers` to get all the stakers
User `batchApplyStakeChange` to apply the stakers change